adhocracy4/polls/assets/PollQuestion.jsx

```diff
@@ -63,7 +71,7 @@ export const PollQuestion = (props) => {
                       id={'id_choice-' + choice.id + '-single'}
                       value={choice.id}
                       checked={checked}
-                      onChange={(event) => { handleSingleChange(event) }}
+                      onChange={(event) => { handleSingleChange(event, choice.is_other_choice, checked) }}
```

Comment:
Do you need the checked there as an argument?

Reply:
oops, yes was a leftover of something i tried before. thanks for the hint